test/e2e/run_as_not_root_test.go

```diff
@@ -23,6 +23,15 @@ func (s *RunAsNonRootSuite) TestRunAsNonRootWorkflow() {
 		WaitForWorkflow(fixtures.ToBeSucceeded)
 }
 
+func (s *RunAsNonRootSuite) TestRunAsNonRootWithOutputParams() {
+	s.Need(fixtures.None(fixtures.Docker, fixtures.K8SAPI, fixtures.Kubelet))
```

Comment:
minor - do you want to include `emissary` in your test? instead maybe just 

```
s.Need(fixtures.PNS)
```

Reply:
The test passes under emissary, and it is a valid use case (I can attest to that!) so I think we want to keep it for emissary to ensure there are no regressions in the future. That is unless you have reasons to explicitly not support this use case for emissary. Please let me know if that's the  case and I will drop it for emissary.